processing/src/main/java/org/apache/carbondata/processing/loading/model/LoadOption.java

```diff
@@ -188,6 +188,8 @@
     optionsFinal.put(CarbonCommonConstants.CARBON_LOAD_MIN_SIZE_INMB,
         Maps.getOrDefault(options, CarbonCommonConstants.CARBON_LOAD_MIN_SIZE_INMB,
             CarbonCommonConstants.CARBON_LOAD_MIN_SIZE_INMB_DEFAULT));
+
+    optionsFinal.put("range_column", Maps.getOrDefault(options, "range_column", null));
```

Comment:
Does makeCreateTableString of makeCreateTableString need add "range_column" ?

Reply:
now it only try to support load data command